grafast/grafserv/src/servers/h3/v1/index.ts

```diff
@@ -1,8 +1,8 @@
-import type { IncomingMessage, Server as HTTPServer } from "node:http";
-import type { Server as HTTPSServer } from "node:https";
-import type { Duplex } from "node:stream";
 import { PassThrough } from "node:stream";
 
+//@ts-expect-error type imports.
+import type { Hooks, Peer } from 'crossws';
```

Comment:
Yes, the lib is typed, but it seems types ar exported as module by default, and grafserv is CommonJS.

I have this error else : 
`The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("crossws")' call instead.
  To convert this file to an ECMAScript module, change its file extension to '.mts' or create a local package.json file with '{ "type": "module" }'.`
Maybe a wrong configuration of my dev environment somewhere ??
  
Because we import only types, this can be ignored.

Reply:
Okay, please use this for now then:

```ts
type Hooks = import('crossws').Hooks;
type Peer = import('crossws').Peer;
```

Reply:
Same error with your proposal (in french):
<img width="691" alt="image" src="https://github.com/user-attachments/assets/5e07ef27-b21b-44f7-925f-ffa885089b5a">

Reply:
Okay; no worries.